### PR TITLE
Initialize connection details from environment-variables

### PR DESCRIPTION
**Pull request by barbelity**

### CODE REVIEW

File: src/main/java/com/jfrog/ide/idea/configuration/XrayServerConfigImpl.java [hunk 1]
```diff
@@ -197,10 +199,30 @@ void setPassword(String password) {
...
         setUsername(credentials.getUserName());
         setPassword(credentials.getPasswordAsString());
     }
 
+    /**
+     * Initialize connection details from environment variables.
+     * Minimal required details are url and username.
+     */
+    public void initConnectionDetailsFromEnv() {
+        String urlEnv = System.getenv(URL_ENV);
```
Comment (yahavi) [inline src/main/java/com/jfrog/ide/idea/configuration/XrayServerConfigImpl.java:214]: This approach should work on 100% of the cases. See explanation above [getEnvironmentMap()](https://github.com/JetBrains/intellij-community/blob/webstorm/193.5233.13/platform/util/src/com/intellij/util/EnvironmentUtil.java#L128).
```suggestion
        String urlEnv = EnvironmentUtil.getValue(URL_ENV);
```

File: src/main/java/com/jfrog/ide/idea/configuration/XrayServerConfigImpl.java [hunk 2]
```diff
@@ -197,10 +199,30 @@ void setPassword(String password) {
...
+        String usernameEnv = System.getenv(USERNAME_ENV);
```
Comment (yahavi) [inline src/main/java/com/jfrog/ide/idea/configuration/XrayServerConfigImpl.java:215]:
```suggestion
        String usernameEnv = EnvironmentUtil.getValue(USERNAME_ENV);
```

File: src/main/java/com/jfrog/ide/idea/configuration/XrayServerConfigImpl.java [hunk 3]
```diff
@@ -197,10 +199,30 @@ void setPassword(String password) {
...
+        String passwordEnv = System.getenv(PASSWORD_ENV);
```
Comment (yahavi) [inline src/main/java/com/jfrog/ide/idea/configuration/XrayServerConfigImpl.java:216]:
```suggestion
        String passwordEnv = EnvironmentUtil.getValue(PASSWORD_ENV);
```

File: src/main/java/com/jfrog/ide/idea/configuration/XrayServerConfigImpl.java [hunk 4]
```diff
@@ -197,10 +199,30 @@ void setPassword(String password) {
...
+        if (StringUtils.isBlank(urlEnv) || StringUtils.isBlank(usernameEnv)) {
+            return;
+        }
+        setUrl(urlEnv);
+        setUsername(usernameEnv);
+        setPassword(passwordEnv);
+        addCredentialsToPasswordSafe();
```
Comment (yahavi) [inline src/main/java/com/jfrog/ide/idea/configuration/XrayServerConfigImpl.java:223]: I'm not sure that it is necessary in this case. We won't want to bother the user with password prompts when it is not really necessary.
```suggestion
```

Comment (barbelity) [reply src/main/java/com/jfrog/ide/idea/configuration/XrayServerConfigImpl.java:223]: We decided to read the config-details once, and store them for the user.
Then, in future uses if the configuration is already set, we are not reading env-vars again.

**Pull request merged**